modules/ROOT/pages/intro-platform-hosting.adoc

```diff
@@ -0,0 +1,146 @@
+= Anypoint Platform Hosting Options
+
+You can host different parts of Anypoint Platform in different locations, either in the cloud or in a physical data center. The best hosting option for your Anypoint Platform environment depends on your security, data privacy, and legal requirements. Different hosting options enable you to control the location where your and your customer's data is stored, processed, and managed. The data remains segregated within the physical location required by your companies IT policies and requirements.
+
+== Control Plane vs Runtime Plane
+
+To understand the different hosting options, it is important to understand the concepts of control plane and runtime plane within the context of Anypoint Platform.
+
+image::intro-control-vs-runtime-planes.png[]
+
+* Control plane
++
+Control plane refers to the components of the Anypoint Platform architecture you use to design, deploy, and manage APIs and Mule applications. The control plane contains product features and components that are part of the Anypoint Design Center, Anypoint Management Center and Exchange.
+
+* Runtime plane
++
+Runtime plane refers to the components of the Anypoint Platform architecture where your APIs and Mule applications are deployed and made available to your users. The runtime plane includes the Mule runtime server and supporting services.
+
+=== Which Hosting Option is Right for You?
+
+Anypoint Platform is designed to enable the control plane and runtime plane to be hosted independently. Depending on your requirements, you can choose from the following: 
```

Comment:
It's the components within the planes, but the planes are really just wrappers or groupings of similarly functioning components of the Anypoint Platform.

Reply:
They are really there to avoid having to repeat Design Center, Management Center, Exchange everytime. Also, according to Sundip these are industry standard terms. They are also terms that PM and support use with customers, but our use should be reserved specifically for this context. Elsewhere we should be more specific and use (if possible) what appears in the UI.